packages/carbon-graphs/src/js/controls/Gantt/helpers/actionHelpers.js

```diff
@@ -255,13 +248,13 @@ const renderDataPointPath = (scale, config, path, dataPoint, index) => path.appe
  * @param {d3.selection} canvasSVG - d3 html element of the canvas
  * @returns {object} - d3 append object
  */
-const drawActionDataPoints = (scale, config, canvasSVG) => canvasSVG
+const drawActionDataPoints = (scale, config, canvasSVG, legendSVG) => canvasSVG
```

Comment:
nit: JS description not updated with the new parameter `legendSVG`

Reply:
updated here https://github.com/cerner/terra-graphs/pull/18/commits/30019d1a1afbecf20ff1b8476b29e225df1cc68b